### ISSUE TITLE
angular 9 

### ISSUE THREAD

**Issue by RBleyenberg** [briebug/ngrx-auto-entity#108]
Howdy,

When can we see the support for angular 9 rolling in ....

Thanks upfront

**Comment by jrista** [comment]
We are looking into it. We also are looking into NgRx 9, as well as potential RxJs 7 support as well, once those are released. We do try to maintain backwards compatibility, so as no to alienate users that are on NgRx8/Angular8 and not yet able to upgrade, so we have to be a bit more careful about these things.

We'll keep you posted!

**Comment by jrista** [comment]
I thought I would provide an update on this. We have used NgRx Auto-Entity in Angular 9 projects, and so far have not identified any issues. We have been testing with a forthcoming version, 0.5, that has had a variety of enhancements, however it is still the same general library as the currently released 0.4, with the same kinds of hooks into the injector. 

We do not foresee any issues using ngrx-auto-entity @~0.4 or @~0.5 with Angular 9 projects. 

We have also done some preliminary testing with Angular 10. We have encountered some issues with the ngcc module compiler. We are looking into this and are planning to release any necessary resolutions in version 0.6 or 0.7, which will be coming soon after 0.5.
